doc/source/conf.py

```diff
@@ -94,9 +94,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'1.0.0'
+version = u'2.0.0'
```

Comment:
Is there some way we can pull this in from the `VERSION` file rather than having to remember to edit it in two places?

Reply:
Or, to rephrase, any reason NOT to just replace this with (some kind of ) `open("../../VERSION").read()`?

Reply:
Easier to keep it this way, otherwise we will have to update a release somehow as well. We could create a ticket for that.

Reply:
I'm not sure I understand what you mean. What would we have to update as well ?
Would you be up for opening a ticket for this - you probably have a better understanding what it's got to be about?

Reply:
See https://github.com/GPflow/GPflow/issues/1371